src/main/java/blusunrize/immersiveengineering/common/data/Recipes.java

```diff
@@ -77,14 +80,11 @@ protected void registerRecipes(@Nonnull Consumer<IFinishedRecipe> out)
 				if(IEBlocks.Metals.ores.containsKey(metal))
 				{
 					Block ore = IEBlocks.Metals.ores.get(metal);
-					CookingRecipeBuilder.smeltingRecipe(Ingredient.fromItems(ore), ingot, metal.smeltingXP, 20)
-							.addCriterion("has_"+toPath(ore), hasItem(ore))
-							.build(out);
+					addStandardSmeltingBlastingRecipe(ore, ingot, metal.smeltingXP, out);
 				}
 			}
-			CookingRecipeBuilder.smeltingRecipe(Ingredient.fromItems(dust), ingot, 0, 20)
-					.addCriterion("has_"+toPath(dust), hasItem(dust))
-					.build(out, toRL(toPath(ingot)+"_from_dust"));
+			addStandardSmeltingBlastingRecipe(dust, ingot, 0, out, "_from_dust");
+//			addStandardSmeltingBlastingRecipe(dust, ingot, metal.smeltingXP, out, "_from_dust"); //TODO: remove this, if 0 XP on dust is intentional. this bugs out because the alloys do not have metal.smeltingXP
```

Comment:
It's probably intentional to prevent XP loops by repeatedly crushing and smelting some metal.

Reply:
Yea, my guess, too. Still left it there just in case it wasn't and still needed xp added for the alloy metals.